infrastructure-playbooks/rolling_update.yml

```diff
@@ -110,10 +101,21 @@
         enabled: yes
       when: is_systemd
 
-    - name: select a running monitor
+    - name: set mon_host_count
+      set_fact: mon_host_count={{ groups.mons | length }}
+
```

Comment:
can we prompt a message if `mon_host_count=1`? Something like: "are you sure you want to continue?"